manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -131,7 +131,7 @@ spec:
             failureThreshold: 30
             httpGet:
               path: /healthz/ready
-              port: 15020
+              port: 15021
```

Comment:
opinions on the port number are welcome

Reply:
Off by 1 is always a good choice :)

Reply:
This would require folks to change their health check script to pick up the new port.